torch/export/_trace.py

```diff
@@ -742,6 +747,8 @@ def forward(self, *args, **kwargs):
             example_inputs=(args, kwargs),
             constants=ep_non_strict.constants,
         )
+        ep._flags = flags
```

Comment:
what's the flags for?

Reply:
ok nvm it doesn't quite work lol. I use a global instead.